sorc/link_fix.sh

```diff
@@ -21,6 +21,8 @@ elif [ ${target} == "orion" ]; then
   FIXROOT=/work/noaa/hwrf/noscrub/hafs-fix-files/hafs-${FIXversion}-fix/fix
 elif [ ${target} == "jet" ]; then
   FIXROOT=/lfs4/HFIP/hwrf-data/hafs-fix-files/hafs-${FIXversion}-fix/fix
+elif [[ $target =~ gaea ]]; then
+  FIXROOT=/lustre/f2/dev/Samuel.Trahan/hafs/hafs-fix-files/hafs-${FIXversion}-fix/fix
```

Comment:
Is there a plan to move this out of your personal directory?

Reply:
Do we have somewhere else to put it?

Reply:
We should be able to put it in the role.epic space somewhere.

Reply:
@mark-a-potts -

Please copy this directory to a communal area:

/lustre/f2/dev/Samuel.Trahan/hafs/hafs-fix-files

and let me know when it is there. I'll update the link_fix.sh.

Reply:
I don't want to wait any longer. I'll merge this now. If you provide me a directory later, I'll update the script.